src/main/resources/com/fulcrumgenomics/bam/ErrorRateByReadPosition.R

```diff
@@ -0,0 +1,59 @@
+#################################################################################
```

Comment:
We don't execute the script directly, we invoke Rscript and pass it, so it wouldn't do anything.